app/controllers/api/v1/users_controller.rb

```diff
@@ -120,4 +120,24 @@ def update
     standard_update(User, current_human_user.id)
   end
 
-end
\ No newline at end of file
+  ###############################################################
+  # find_unclaimed
+  ###############################################################
+
+  api :POST, '/unclaimed/:email', 'Finds or creates a user by email address.'
```

Comment:
Do we need a `/users/find` endpoint?  Since the purpose of this is to allow applications to create an account using an email, shouldn't we name it `/users/create/email/foo@test.com`  The endpoint would continue to be implemented in terms of `find-or-create` where it would only return an existing id and not create a duplicate.